k8s/test/e2e/autogenerated-helm-chart/e2e.yaml

```diff
@@ -25,9 +25,9 @@ setup:
       command: |
         helm dep update charts/vineyard-operator
         helm install vineyard-operator charts/vineyard-operator \
+            --create-namespace \
             --namespace vineyard-system \
-            --set controllerManager.manager.image.repository=localhost:5001/vineyard-operator \
-            --create-namespace
+            --set manager.image.repository=localhost:5001/vineyard-operator
```

Comment:
The field of image is wrong, refer to https://github.com/v6d-io/v6d/blob/main/charts/vineyard-operator/values.yaml#L23